richeditor-compose/src/commonMain/kotlin/com/mohamedrejeb/richeditor/model/RichTextState.kt

```diff
@@ -569,21 +583,41 @@ class RichTextState internal constructor(
         )
     }
 
+    @Deprecated("Use toggleUnorderedList() instead to add or remove unordered list")
     fun removeUnorderedList() {
         val paragraph = getRichParagraphByTextIndex(selection.min - 1) ?: return
+        removeUnorderedList(paragraph)
```

Comment:
Same as above, this shouldn't be deprecated.

Reply:
ready